test/unstable/arrayFuncs.chpl

```diff
@@ -0,0 +1,11 @@
+// We were considering deprecating these but have since pulled back the the
```

Comment:
Typo: double "the"

Reply:
Fixed in 3a0f2ca925bfe51dc488a4e997ec58d78a930989